### PR TITLE
Added Responsive Profile Card 

### PR DESCRIPTION
**Pull request by Sulagna-Dutta-Roy**
Fixes: #155 
Tech stack: Html5,Css,Javascript

Profile Card
Profile Card1

### CODE REVIEW

**Comment by Sulagna-Dutta-Roy** [comment]
@Abirpal202049 @nandini7637 kindly review this PR

File: Cards/Personal Profile Card/code/style.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Josefin Sans',sans-serif;
+}
+body{
+    background: linear-gradient(to right, #fefffe, rgb(185, 182, 182));
+}
```
Comment (Abirpal202049) [inline Cards/Personal Profile Card/code/style.css:9]: target the body by class or id

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]

File: Cards/Personal Profile Card/code/style.css (hunk 1 above)
Comment (Abirpal202049) [reply Cards/Personal Profile Card/code/style.css:9]: Else everything is Ok

**Comment by Sulagna-Dutta-Roy** [comment]
Done the changes. Kindly check

**Review by Abirpal202049 — APPROVED** [review]

**Pull request merged**